--- matplotlib-1.4.1/setupext.py	Tue Oct 21 09:47:33 2014
+++ matplotlib-1.4.1/setupext_new.py	Tue Oct 21 09:49:38 2014
@@ -991,7 +991,7 @@
                 'lib/freetype2/include/freetype2'],
             default_library_dirs=[
                 'freetype2/lib'],
-            default_libraries=['freetype', 'z'])
+            default_libraries=[])
 
 
 
@@ -1039,7 +1039,7 @@
             ]
         ext = make_extension('matplotlib._png', sources)
         pkg_config.setup_extension(
-            ext, 'libpng', default_libraries=['png', 'z'],
+            ext, 'libpng', default_libraries=[],
             alt_exec='libpng-config --ldflags')
         Numpy().add_flags(ext)
         CXX().add_flags(ext)
