diff -urN pyside-1.2.2/CMakeLists.txt pyside-1.2.2_new/CMakeLists.txt
--- pyside-1.2.2/CMakeLists.txt	Tue Jun 17 15:53:18 2014
+++ pyside-1.2.2_new/CMakeLists.txt	Tue Jun 17 15:57:16 2014
@@ -5,11 +5,15 @@
 cmake_minimum_required(VERSION 2.6)
 set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/Macros/
                       ${CMAKE_MODULE_PATH})
+find_package(PythonLibs 2.6)
+find_package(PythonInterp)
 find_package(Shiboken 1.2.0 REQUIRED)
 find_package(Qt4 4.5.0 REQUIRED)
 find_file(GL_H "gl.h" PATH_SUFFIXES "GL")
 include(FindQt4Extra)
 
+set(CMAKE_DEBUG_POSTFIX "_d")
+
 set(XVFB_EXEC "")
 option(USE_XVFB "Uses xvfb-run with the unit tests to avoid QtGui tests popping windows on the screen." FALSE)
 if(USE_XVFB)
@@ -29,7 +33,7 @@
 endif()
 
 if(MSVC)
-    set(CMAKE_CXX_FLAGS "/Zc:wchar_t- /GR /EHsc /DNOCOLOR /DWIN32 /D_WINDOWS /D_SCL_SECURE_NO_WARNINGS")
+    set(CMAKE_CXX_FLAGS "/Zc:wchar_t- /GR /EHa /DNOCOLOR /DWIN32 /D_WINDOWS /D_SCL_SECURE_NO_WARNINGS")
 else()
     if(CMAKE_HOST_UNIX AND NOT CYGWIN)
         set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -fvisibility=hidden -Wno-strict-aliasing")
