diff -r 8eab90908f2a setup.py
--- a/setup.py	Tue Dec 07 00:27:14 2010 +0100
+++ b/setup.py	Fri Oct 17 18:20:13 2014 -0700
@@ -122,12 +122,12 @@
                   "interfaces" + os.sep + "%s.i"
 
     SOGUI = ['soqt', 'soxt', 'sogtk', 'sowin']
-    MODULES = {'coin'      : ('_coin',      'coin-config',      'pivy.'),
-               'simvoleon' : ('_simvoleon', 'simvoleon-config', 'pivy.'),
-               'soqt'      : ('gui._soqt',  'soqt-config',      'pivy.gui.'),
-               'soxt'      : ('gui._soxt',  'soxt-config',      'pivy.gui.'),
-               'sogtk'     : ('gui._sogtk', 'sogtk-config',     'pivy.gui.'),
-               'sowin'     : ('gui._sowin', 'sowin-config',     'pivy.gui.')}
+    MODULES = {'coin'      : ('_coin',      'coin-config',      'pivy.')}
+    #           'simvoleon' : ('_simvoleon', 'simvoleon-config', 'pivy.'),
+    #           'soqt'      : ('gui._soqt',  'soqt-config',      'pivy.gui.'),
+    #           'soxt'      : ('gui._soxt',  'soxt-config',      'pivy.gui.'),
+    #           'sogtk'     : ('gui._sogtk', 'sogtk-config',     'pivy.gui.'),
+    #           'sowin'     : ('gui._sowin', 'sowin-config',     'pivy.gui.')}
 
     SUPPORTED_SWIG_VERSIONS = ['1.3.31', '1.3.33', '1.3.35', '1.3.40']
     SWIG_VERSION = ""
